app/jobs/diego/sync.rb

```diff
@@ -28,7 +27,6 @@ def perform
             elapsed_ms = ((finish - start) * 1000).round
 
             @statsd.timing('cc.diego_sync.duration', elapsed_ms)
-            @prometheus_updater.report_diego_cell_sync_duration(elapsed_ms)
```

Comment:
This is being emitted on the scheduler VM. Therefore it can't be collected because there is no web server running.